tests/test_is_output_correct.py

```diff
@@ -24,9 +24,15 @@
         (5, 5.0),
         ("1/2", 0.5),
         ("\\frac{1}{2}", 0.5),
+        ("\\frac {1}{2}", 0.5),
```

Comment:
Please remove all of these test cases - I've ported them to another branch where they will be supported (except the two for which I left comments).